### PR TITLE
centml cli support

### PR DESCRIPTION
**Pull request by anandj91**
1. Using click package instead of argparse
2. Using https://github.com/CentML/platform_api_python_client for making requests to platform api
3. CCluster commands ls, get, delete, pause, and resume (create is coming)
4. jwt authentication support with automatic refresh on token expiry

### CODE REVIEW

File: centml/sdk/auth.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    ).json()
+
+    with open(Config.centml_cred_file, 'w') as f:
+        json.dump(cred, f)
+
+    return cred
+
+
+def store_centml_cred(token_file):
+    with open(token_file, 'r') as f:
```
Comment (destefy) [inline centml/sdk/auth.py:28]: there are no checks to see if the file exists. Python will of course throw an error, but maybe we want to check fail gracefully with a "file does not exist" message

File: centml/sdk/auth.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        refresh_centml_token(refresh_token)
+
+
+def load_centml_cred():
+    cred = None
+
+    if os.path.exists(Config.centml_cred_file):
+        with open(Config.centml_cred_file, 'r') as f:
+            cred = json.load(f)
```
Comment (destefy) [inline centml/sdk/auth.py:40]: for now are we assuming that the content of centml_cred_file is always valid and that we don't have to check the content?
also, json.load can fail if the file it reads isn't json parsable, so maybe a try-except block is good?

Comment (anandj91) [reply centml/sdk/auth.py:43]: centml_cred_file is set by us. so normally it should always be valid and parsable. the only case is when the file is corrupt or the user intentionally modify it. but we can handle these cases later.

Comment (destefy) [reply centml/sdk/auth.py:43]: makes sense :+1:

**Review by destefy — APPROVED** [review]

**Pull request merged**